### DISCUSSION TITLE
Support for Ledger Hardware Wallets.

### DISCUSSION THREAD

**Discussion by pablopgus** [Chia-Network/chia-blockchain#7997] [Ideas]
Is this on the table?

**Comment by diaconom** [comment]
I second this request-> Ledger hardware support for HNT.

**Comment by hoffmang9** [comment]
We are waiting for Ledger to support BLS signatures in G2. They have implemented G1 and are working on it.

**Reply by dustingrady** [reply]
Hm, it seems a couple days ago someone closed an issue (requesting ledger support for Chia) stating Chia needs to develop these capabilities before it can be added (https://github.com/Chia-Network/chia-blockchain/discussions/7997).

It sounds like Ledger needs to support BLS sigs in G2 before that is possible?

**Reply by hoffmang9** [reply]
There are refactoring changes already underway that will modify private key management to support hardware wallets.

**Reply by dustingrady** [reply]
Got it. I also just saw your comments in the Trello board. Thanks again for all your work!

**Reply by jmz-bit** [reply]
Any progress regarding Ledger support BLS signatures in G2?

**Comment by williambanfield** [comment]
@hoffmang9 do you a link to information documenting Ledger's support for G1 / timeline for G2? I can't seem to find any. Thanks!

**Reply by IdaTucker** [reply]
In their [developers documentation](https://developers.ledger.com/docs/embedded-app/crypto-api/ox__ec_8h/#enumerations), they list the available curves, and only CX_CURVE_BLS12_381_G1 is cited.
